### PR TITLE
Fix: Fix windows paths for executables

### PR DESCRIPTION
**Pull request by eakmanrq**
Prior to this change if a path was generated using a Windows computer it would generate a different fingerprint than a linux/macos computer. This normalizes them.

### CODE REVIEW

**Comment by tobymao** [comment]
this is a breaking change and requires a migration trigger

File: sqlmesh/migrations/v0016_fix_windows_path.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        parsed_snapshot = json.loads(snapshot)
+        model = parsed_snapshot["model"]
+        python_env = model.get("python_env")
+        if not python_env:
+            continue
+        for py_definition in python_env.values():
+            path = py_definition.get("path")
+            if not path:
+                continue
+            py_definition["path"] = path.replace("\\", "/")
```
Comment (eakmanrq) [inline sqlmesh/migrations/v0016_fix_windows_path.py:27]: @tobymao This assumes that any backslash in the path is part of a windows path.

**Review by tobymao — APPROVED** [review]

**Comment by tobymao** [comment]
do we have this problem anywhere else?

**Comment by eakmanrq** [comment]
> do we have this problem anywhere else?

Not that I can see

**Pull request merged**